beamerthemeuulm.sty

```diff
@@ -149,11 +160,19 @@
 % table of contents
 % ---------------------------------------------
 \AtBeginSection{
-    \begin{frame}{\thesection.~\insertsection}
+    \ifsectiontitleslide
```

Comment:
We could move this if to cover the complete `\AtBeginSection` command, removing the hook completely if not needed.

Reply:
You are right, i will change this.

Reply:
I am happy with that. However, you could reduce the indentation level of the `\ifsectionoverview`-block by one now or increase the indentation level of the `\AtBeginSection{`..`}`-lines.

Reply:
I changed this in my latest commit.